src/applications/disability-benefits/4142/tests/00.4142-entry.e2e.spec.js

```diff
@@ -0,0 +1,14 @@
+const Timeouts = require('../../../../platform/testing/e2e/timeouts');
+const E2eHelpers = require('../../../../platform/testing/e2e/helpers');
+const manifest = require('../manifest.json');
+
+module.exports = E2eHelpers.createE2eTest((client) => {
+  client
+    .url(`${E2eHelpers.baseUrl}/disability-benefits/private-medical-records`)
+    .waitForElementVisible('body', Timeouts.normal)
+    .axeCheck('.main');
+
+  client.end();
+});
+
+module.exports['@disabled'] = !manifest.production || __BUILDTYPE__ !== 'production';
```

Comment:
Interesting. I've never seen this before. (And of course a quick Google didn't yield anything useful.) Does `module.exports['@disabled'] = true` mean that it won't _actually_ export anything?

Reply:
This was generated from the most recent version of the form builder.

Reply:
Ahh, okay. It looks like it's a [Nightwatch thing](https://github.com/nightwatchjs/nightwatch-docs/blob/master/guide/running-tests/disabling-tests.md#disabling-tests).